website/source/docs/config/index.html.md

```diff
@@ -739,6 +742,41 @@ LANGUAGE plpgsql;
 
 More info can be found in the [PostgreSQL documentation](http://www.postgresql.org/docs/9.4/static/plpgsql-control-structures.html#PLPGSQL-UPSERT-EXAMPLE):
 
+#### Backend Reference: Cassandra (Community-Supported)
+
+The Cassandra backend has the following options:
+
+  * `hosts` (optional) – Comma-separated list of Cassandra hosts to connect to.
+    Defaults to `"127.0.0.1"`.
+
+  * `keyspace` (optional) – Cassandra keyspace to use. Defaults to `"vault"`.
+
+  * `table` (optional) – Table within the `keyspace` in which to store data.
+    Defaults to `"entries"`.
+
+  * `consistency` (optional) – Consistency level to use when reading/writing data
+    in Cassandra. If set, must be one of `"ANY"`, `"ONE"`, `"TWO"`, `"THREE"`, `"QUORUM"`,
+    `"ALL"`, `"LOCAL_QUORUM"`, `"EACH_QUORUM"`, or `"LOCAL_ONE"`. Defaults to `"LOCAL_QUORUM"`.
+
+You need to ensure the keyspace and table exist in Cassandra:
+
+```cql
+CREATE KEYSPACE "vault" WITH REPLICATION = {
```

Comment:
I quoted it because [`entries` is a keyword in CQL](http://docs.datastax.com/en/cql/3.3/cql/cql_reference/keywords_r.html). I can change the name of the default table, or keep it quoted. I felt that quoting was the better solution here, I don't think that users would necessarily expect this to be case-insensitive.